Chapters/1-8-Interlogue.md

```diff
@@ -0,0 +1,45 @@
+## Interlogue
+
+The magistrate re-entered the room while the cell guard held Kipnu's hands behind his back. In the light, Kipnu could see that the magistrate's eyes were wholly uninterested in his job. They were disappointed in Kipnu, but not surprised.
+
+"Kipnu Um Heiko. I find you guilty of assault on Fajro Um Holrag and interruption of his consciousness. The sentence shall be a period of two La'To in Ulukoma. Do you understand?"
+
+"Do I have a choice?"
+
+"Not really. Rante, an escort will be here just after first lunch to escort Kipnu to the docks."
+
+"What do I do with him 'til then?"
+
+"Do what you do best. Lock him up and take a nap."
+
+"Hah, you got it."
+
+He turned to Kipnu and said, "Kipnu. I hope that one day you might join us to prepare for the trials ahead. Until then, may Ulukoma quench whatever poison plagues your consciousness."
+
+"You think people actually listen to your self-righteous trash?"
+
+The magistrate simply sighed and walked out the door. Rante shoved Kipnu back into the dark room of cells and locked him in a cell next to Liffa.
+
+"Fajro, I hear the hero is going with you. When do you want me to call an escort?"
+
+"We won't be needing an escort."
+
+"You sure? Liffa broke into the senate. and I'm sure people might certainly try what Kipnu did."
+
+"I'm positive. The trouble is over."
+
+"Suit yourself. You're free to take hae whenever. Here's the key, and I've got armchains in the other room if you need them."
+
+"Thank you, sir. That will be all," said Fajro, accepting the key, and Rante rushed back to his desk.
+
+Kipnu sat still and silent while I shifted the corner of hae cell closest to him and stuck my hand out of the little window, tapping on the wall for him to do the same. He shifted in his cell and then I felt his hand touch mine. It was cold and at first I wasn't sure if it was him, but I could feel the textures and shapes were the same and it broke my heart.
```

Comment:
Wait?! How did we switch back to Lif's POV? I think you are missing `~*~` on ln 22.